ain/utils.py

```diff
@@ -0,0 +1,114 @@
+import re
+import json
+import time
+from typing import Any, Union
+from Crypto.Hash import keccak as _keccak
+from coincurve import PrivateKey
+from ain.types import ECDSASignature, TransactionBody
+
+
+def getTimestamp() -> int:
+    """
+    Gets a current timestamp.
+    """
+    return int(time.time() * 1000)
+
+
```

Comment:
isn't just one blank line enough?

Reply:
I used that `black` to manage the code style, and that was auto-corrected by it. If not using it is considered better, I'll code manually.

Reply:
Thanks for the explanation. If possible I'd recommend to turn off auto formatter unless it helps a lot for tedious formatting jobs like trailing space removal.

Reply:
DONE